packages/tldraw/src/lib/shapes/draw/DrawShapeTool.ts

```diff
@@ -6,6 +6,7 @@ import { Idle } from './toolStates/Idle'
 export class DrawShapeTool extends StateNode {
 	static override id = 'draw'
 	static override initial = 'idle'
+	static override isLockable = false
```

Comment:
huh, didn't realize Draw and Text are not lockable...
actually, why is that? for my understanding

Reply:
Draw tool is a bit special since we don't exit it at all (even though it is not lockable). Makes sense since when you start drawing you probably want to draw multiple shapes. Not sure about text, has been like that since I can remember.

cc @steveruizok

Reply:
Yep, the draw tool predates tool lock.

Reply:
but whyyy though. why is Text not lockable? i don't think there's a good reason to not let it be lockable.